packages/server/src/utils/buildChatflow.ts

```diff
@@ -344,7 +344,8 @@ export const utilBuildChatflow = async (req: Request, socketIO?: Server, isInter
             memoryType,
             sessionId,
             createdDate: userMessageDateTime,
-            fileUploads: incomingInput.uploads ? JSON.stringify(fileUploads) : undefined
+            fileUploads: incomingInput.uploads ? JSON.stringify(fileUploads) : undefined,
+            leadEmail: incomingInput.leadEmail
```

Comment:
maybe i missed this, but I dont see anywhere that we are sending the request that has `leadEmail` in the body.

After saved the leads, and proceed to chat, I can see a new lead has been created on the database table `lead`, but for `chat_message` table, the `leadEmail` is still empty

Reply:
Chat embedded does work